pipenv/project.py

```diff
@@ -757,7 +757,7 @@ def create_pipfile(self, python=None):
                 required_python = self.which("python")
         version = python_version(required_python) or PIPENV_DEFAULT_PYTHON_VERSION
         if version and len(version) >= 3:
-            data[u"requires"] = {"python_version": version[: len("2.7")]}
+            data[u"requires"] = {"python_version": version.rsplit(".", 1)[0]}
```

Comment:
```suggestion
            data[u"requires"] = {"python_version": ".".join(version.split(".")[:2])}
```
This should be the desired implementation.

Reply:
Thanks for reviewing. I have revised the commit.